source/acquaman/SGM/SGMXASScanActionController.cpp

```diff
@@ -214,19 +216,15 @@ AMAction3* SGMXASScanActionController::createInitializationActions(){
 
 	AMListAction3 *initializationStage1 = new AMListAction3(new AMListActionInfo3("SGM XAS Initialization Stage 1", "SGM XAS Initialization Stage 1"), AMListAction3::Parallel);
 
-	tmpControl = SGMBeamline::sgm()->grating();
-	AMControlInfo gratingSetpoint = tmpControl->toInfo();
-	gratingSetpoint.setValue(configuration_->fluxResolutionGroup().controlNamed(tmpControl->name()).value());
-	moveActionInfo = new AMControlMoveActionInfo3(gratingSetpoint);
-	moveAction = new AMControlMoveAction3(moveActionInfo, tmpControl);
-	initializationStage1->addSubAction(moveAction);
+    tmpControl = SGMBeamline::sgm()->grating();
```

Comment:
Can you check your tab spacing?  If it's not set correctly it will skew the llines added/removed in the commits.